game_loop_entity.py

```diff
@@ -75,8 +121,6 @@ def run(self):
             initial_pos, final_pos = self.gather_user_input()
             self.led_interface.stop_all()
 
-
-
             # Give to chess lib
             self.give_to_chess_library(initial_pos, final_pos)
```

Comment:
This method will raise a `ValueError` if the move given by the user is incorrect. You should:

- 1. Check that `give_to_chess_library` does not raise a `ValueError`
    - If it does, tell the user the move they made is invalid
    - Somehow ask them for input again